shinyswatch/_get_theme_deps.py

```diff
@@ -4,9 +4,47 @@
 
 from htmltools import HTMLDependency
 
+# from shiny.ui._html_deps_external import bootstrap_deps_suppress
+from shiny._versions import bootstrap as shiny_bootstrap_version
+
 from ._assert import assert_theme
 from ._bsw5 import BSW5_THEME_NAME, bsw5_version
-from ._shiny import base_dep_version, bs5_path, bs_dep_no_files
+
+bs5_path = os.path.join(os.path.dirname(__file__), "bs5")
+
+
+def suppress_shiny_bootstrap() -> list[HTMLDependency]:
+    return [
+        # shiny > 0.8.1 will (likely) split bootstrap into separate js/css deps
+        HTMLDependency(
+            name="bootstrap-js",
+            version=shiny_bootstrap_version + ".9999",
+        ),
+        HTMLDependency(
+            name="bootstrap-css",
+            version=shiny_bootstrap_version + ".9999",
+        ),
+        # shiny <= 0.8.1 loads bootstrap as a single dep
+        HTMLDependency(
+            name="bootstrap",
+            version=shiny_bootstrap_version + ".9999",
+        ),
+        # Disable ionRangeSlider
+        # TODO: Remove this when ionRangeSlider no longer requires Sass for BS5+
+        HTMLDependency(
+            name="preset-shiny-ionrangeslider",
+            version="9999",
+        ),
+    ]
+
+
+def dep_shinyswatch_bootstrap_js() -> HTMLDependency:
+    return HTMLDependency(
+        name="shinyswatch-js",
+        version=bsw5_version,
+        source={"package": "shinyswatch", "subdir": bs5_path},
+        script={"src": "bootstrap.bundle.min.js"},
+    )
```

Comment:
Should this only be included for `shiny <= 0.8.1`?

Reply:
No, this PR still removes the Bootstrap JS dependency. We'll take that out in a follow up PR once shiny has some run-time Bootstrap compatibility checks. Then we'll require latest shiny in shinswatch.